### PR TITLE
Introduce extension method for VisualTreeHelper to find the current Frame for navigation.

### PR DESCRIPTION
**Pull request by bruzkovsky**
See: #222

### CODE REVIEW

File: Source/ZXing.Net.Mobile.WindowsUniversal/project.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.0.0"
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.0.0",
+    "WriteableBitmapEx": "1.5.0"
```
Comment (Redth) [inline Source/ZXing.Net.Mobile.WindowsUniversal/project.json]: Not sure why you added this nuget?

**Comment by Redth** [comment]
This looks great, just wondering about the addition of the nuget dependency you added...

**Comment by bruzkovsky** [comment]
Thank you! The nuget is definitely not needed here, I don't know how it slipped in there ... I will fix that in a sec!

**Comment by Redth** [comment]
Thanks very much for this!

**Comment by bruzkovsky** [comment]
Thank you!

**Pull request merged**